rule-audit-client/src/components/capiFeed/CapiResults.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import AppTypes from "AppTypes";
+import { selectors } from "redux/modules/capiContent";
+import { MapStateToProps, connect } from "react-redux";
+import { CapiContent } from "services/Capi";
+
+type IProps = ReturnType<typeof mapStateToProps>;
+
+const CapiResults = ({ content }: IProps) => (
+  <div className="mt-2">
+    {Object.values(content).map(article => (
+      <div className="card mt-2">
+        <div className="card-body">
+          <div className="card-text">{article.webTitle}</div>
+          <div className="badge badge-warning mr-1">2</div>
+          <div className="badge badge-danger">4</div>
```

Comment:
What does `4` mean here? (Same for `2` above).

Reply:
It's a placeholder for different levels of typerighter warning – they don't mean anything at the moment! They're used in an upcoming PR that adds actual validation ... but I appreciate they don't mean much now.